interpolate/src/commonMain/kotlin/Time.kt

```diff
@@ -0,0 +1,19 @@
+package com.juul.krayon.interpolate
+
+import kotlinx.datetime.LocalDateTime
+import kotlinx.datetime.TimeZone
+import kotlinx.datetime.toInstant
+import kotlinx.datetime.toLocalDateTime
+import kotlin.time.Duration
+import kotlin.time.ExperimentalTime
+
+// This functionality doesn't exist in KotlinX DateTime because the specifics of different use-cases
+// affect the desired behavior. See: https://github.com/Kotlin/kotlinx-datetime/issues/66
+
+@OptIn(ExperimentalTime::class)
+internal operator fun LocalDateTime.minus(other: LocalDateTime): Duration =
+    this.toInstant(TimeZone.UTC) - other.toInstant(TimeZone.UTC)
+
+@OptIn(ExperimentalTime::class)
+internal operator fun LocalDateTime.plus(duration: Duration): LocalDateTime =
+    (this.toInstant(TimeZone.UTC) + duration).toLocalDateTime(TimeZone.UTC)
```

Comment:
> I assume your comfortable with Krayon then being opinionated on the matter?

Yup. Since Krayon is only worried about _display_, I think we can get away with ignoring most of the nuance. If an axis happens to cross a daylight-saving time change, I would still expect to see time increasing linearly on the axis and not repeat or skip a value. Measurements that happen to advance monotonically in real-world-time would just not be monotonic along the X axis. Excuse the low quality hand paint, but something like this:

![art](https://user-images.githubusercontent.com/20380774/144523765-93a20968-99f2-4de4-b2c2-52d38d0f476c.png)

> the user may not expect the behavior we've chosen

This is true. But we either make a decision (like I did here), or don't provide the functionality at all (like KotlinX DateTime). All of our interfaces are public, so if a user really needed another option, they can implement it. For users who don't care, the library provides a sane/reasonable default.

Reply:
Such beautiful art. Where can I buy the NFT? 😝